README.md

```diff
@@ -7,6 +7,49 @@ height="42" />
 
 Next-generation WordPress eCommerce Experience for WordPress sites at Newfold Digital.
 
+
+## Module Responsibilities
+
+* Ecommerce module displays `Home or Dashboard` page for Brand plugin, with below options:
+  1. Live site preview for both, live and coming soon mode for WordPress site with view & edit site options
+  2. Next steps to setup your site/store and also link to view completed tasks
```

Comment:
This functionality isn't a critical path, but it is the responsibility of this module. We should add a bit more detail here so it is clear how this works.

Reply:
Added more details about this section.